lib/normalizer.c

```diff
@@ -1247,6 +1254,7 @@ grn_nfkc_normalize_unify_stateless(grn_ctx *ctx,
   i_byte = 0;
   i_character = 0;
   while (current < end) {
+    unsigned char unified_alpha[2];
```

Comment:
* Could you use `alphabet` not `alpha` like other added code?
* Do we need 2 bytes for unified alphabet? I think that 1 byte is enough because an unified alphabet is an ASCII character.

Reply:
> Could you use alphabet not alpha like other added code?
fix: [bba09c9](https://github.com/groonga/groonga/pull/1831/commits/bba09c934036cabbc5adc58fc265470cf69ba1a0) Sure.

> Do we need 2 bytes for unified alphabet? I think that 1 byte is enough because an unified alphabet is an ASCII character.

fix: [9d05eed](https://github.com/groonga/groonga/pull/1831/commits/9d05eed1511ea76d7ca31da08a856bce0ca166c9) I misunderstood the size of ASCII characters.

```ruby
irb(main):001> 'À'.bytesize
=> 2
irb(main):002> 'A'.bytesize
=> 1
```